composer.json

```diff
@@ -25,7 +25,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "5.4-dev"
+            "dev-5.x": "5.4-dev"
```

Comment:
this does not make sense. The version for the 5.x branch is not ``dev-5.x``.
and there is no point aliasing it as it is already comparable